src/libraries/System.IO.FileSystem/tests/File/Create.cs

```diff
@@ -219,9 +220,13 @@ public void LongFileName()
 
         [Fact]
         [PlatformSpecific(CaseSensitivePlatforms)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/51371", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
         public void CaseSensitive()
         {
+            if (RuntimeInformation.RuntimeIdentifier.StartsWith("iossimulator"))
```

Comment:
tvOS simulator will be affected as well:

```suggestion
            if (RuntimeInformation.RuntimeIdentifier.StartsWith("iossimulator") || RuntimeInformation.RuntimeIdentifier.StartsWith("tvossimulator"))
```